### DISCUSSION TITLE
image tracking

### DISCUSSION THREAD

**Discussion by Swapnesh-dev** [google/model-viewer#4858] [Q&A] [answered]
can image tracking be achieved using model-viewer

**Comment by elalish** [comment, marked as answer]
Not currently. PRs welcome!

**Comment by Jatin-Pawar** [comment]
@elalish Does MV support image tracking now?


If not
1) how easy or difficult is it to implement this as per your expereince?
2) Also, could you share some viewpoints/resources to get started with this like high-level view of how it can be done, possible tools/libs to test out and use to achieve this?

PS :Just a brief about me,  I am an AR developer and have experience building with Unity and Unreal and have also done some threejs and WebXR development. Hope it helps to answer :)

**Reply by elalish** [reply]
No, though PRs welcome! I haven't tried to implement it yet, but I'd guess not too bad since it already has support in WebXR and three.js. Maybe this will help: https://www.youtube.com/watch?v=aymZYxjT0AM&ab_channel=FireDragonGameStudio
